app-modules/form/src/Listeners/ClearFormFormSubmissionCountCache.php

```diff
@@ -44,7 +44,7 @@ class ClearFormFormSubmissionCountCache implements ShouldQueue
 {
     public function handle(FormSubmissionCreated $event): void
     {
-        Cache::tags('form-submission-count')
+        Cache::tags('{{form-submission-count}}')
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.